sample/gutters.css.scss

```diff
@@ -0,0 +1,64 @@
+
+@import '../grid-system.scss';
+
+.app {
+	@include vui-gs-container;
+}
+
+.app * {
+	box-sizing: border-box;
+}
+
+.header {
+	@include vui-gs-row;
+	background-color: rgba(0,0,250,0.4);
+	height: 60px;
+}
+
+.stuff-1a {
+	@include vui-gs-span(4);
+	background-color: rgba(250,0,0,0.4);
+	height: 60px;
+}
+
+.stuff-1b {
+	@include vui-gs-span(4);
+	background-color: rgba(250,0,0,0.4);
+	height: 60px;
+}
+
+.stuff-1c {
+	@include vui-gs-span(last 4);
+	background-color: rgba(250,0,0,0.4);
+	height: 60px;
+}
+
+.stuff-2a {
+	@include vui-gs-span(4 wide no-gutter);
```

Comment:
How common do you think using these shorthands will be? Like, did you find them useful when converting copycat? If they're handy AND common I'm all for including them and then dealing with having to write a parser down the road. If they're more edge-case nice-to-have type situations then I'm cool with leaving them out.